model-providers/openai/openai-common/deployment/src/main/java/io/quarkiverse/langchain4j/openai/deployment/OpenAiCommonProcessor.java

```diff
@@ -29,11 +27,6 @@ void indexDependencies(BuildProducer<IndexDependencyBuildItem> producer) {
         producer.produce(new IndexDependencyBuildItem("dev.ai4j", "openai4j"));
     }
 
-    @BuildStep
-    UnremovableBeanBuildItem unremovableBeans() {
-        return UnremovableBeanBuildItem.beanTypes(OpenAiRestApi.AuthProvider.class);
-    }
-
```

Comment:
Has this moved somewhere else?

Reply:
Nice catch, I had forgotten to commit the changed file!

I hope I put it in the right place.

Reply:
Looks great!